django/conf/global_settings.py

```diff
@@ -628,6 +628,15 @@ def gettext_noop(s):
 # The default file storage backend used during the build process
 STATICFILES_STORAGE = "django.contrib.staticfiles.storage.StaticFilesStorage"
 
+# File storage info. If left empty, will default to DEFAULT_FILE_STORAGE and
+# STATICFILES_STORAGE.
+DEFAULT_STORAGE_ALIAS = "default"
+STATICFILES_STORAGE_ALIAS = "staticfiles"
```

Comment:
These aliases should not be configurable. We can keep them as constants but not in settings.

Reply:
Moved to `django/conf/__init__.py`.